qiita_db/test/test_study.py

```diff
@@ -147,11 +145,11 @@ def setUp(self):
             'metadata_complete': True,
             'reprocess': False,
             'number_samples_promised': 27,
-            'emp_person_id': qdb.study.StudyPerson(2),
+            'emp_person_id': 2,
```

Comment:
That's incorrect - note how it is actually the id, the ones that are no longer the id are the principal_investigator and the lab_person